bugbug/bugzilla.py

```diff
@@ -215,3 +221,66 @@ def download_bugs(bug_ids, products=None, security=False):
 
 def delete_bugs(bug_ids):
     db.delete(BUGS_DB, lambda bug: bug["id"] in set(bug_ids))
+
+
+def count_bugs(bug_query_params):
+    bug_query_params["count_only"] = 1
+
+    r = requests.get("https://bugzilla.mozilla.org/rest/bug", params=bug_query_params)
+    r.raise_for_status()
+    count = r.json()["bug_count"]
+
+    return count
+
+
+def get_product_component_csv_report():
+    six_month_ago = datetime.utcnow() - relativedelta(months=6)
+
+    # Base params
+    url_params = {
+        "f1": "creation_ts",
+        "o1": "greaterthan",
+        "x_axis_field": "product",
+        "y_axis_field": "component",
+        "action": "wrap",
+        "ctype": "csv",
+        "format": "table",
+        "v1": six_month_ago.strftime("%Y-%m-%d"),
```

Comment:
Nit: move this right after "o1", so the field, operation, and value of 1 are all close together.